doc/index.md

```diff
@@ -17,6 +17,9 @@ For Developers
 
 -   [Build Objective-C (iOS) WebSocket Clients](dev-objc/o_dev_objc.md)
 -   [Build Android and Java WebSocket Clients](dev-android/o_dev_android.md)
```

Comment:
Under mobile, all clients are together (ws, jms, etc.). Does it make sense to have a JMS Mobile client heading here for clarity? cc: @michaelcretzman